tests/testthat/test-board_connect_url.R

```diff
@@ -0,0 +1,42 @@
+skip_if_not_installed("rsconnect")
```

Comment:
This is needed because the test boards use `connect_has_colorado()`, not because `board_connect_url()` uses the rsconnect package (it does not).